src/components/list-box/_list-box.scss

```diff
@@ -0,0 +1,242 @@
+//-----------------------------
+// List
+//-----------------------------
+
+@import '../../globals/scss/colors';
+@import '../../globals/scss/css--helpers';
+@import '../../globals/scss/layout';
+@import '../../globals/scss/import-once';
+@import '../../globals/scss/vars';
+
+$list-box-width: rem(300px);
+$list-box-height: rem(40px);
+$list-box-inline-height: rem(32px);
+
+@include exports('list-box') {
+  .bx--list-box {
+    position: relative;
+    width: $list-box-width;
+    height: $list-box-height;
+    background-color: rgba($field-01, 0.1);
+  }
+
+  .bx--list-box--disabled {
+    opacity: 0.5;
+  }
+
+  .bx--list-box--disabled,
+  .bx--list-box--disabled .bx--list-box__field,
+  .bx--list-box--disabled .bx--list-box__menu-icon {
+    cursor: not-allowed;
+  }
+
+  .bx--list-box input[role='combobox'] {
+    background-color: inherit;
+    font-weight: 600;
+    outline-offset: 0;
+  }
+
+  .bx--list-box input[role='combobox']::placeholder {
+    font-weight: 400;
+  }
+
+  .bx--list-box--disabled input[role='combobox'] {
+    opacity: 1;
+  }
+
+  .bx--list-box__menu {
+    @include box-shadow();
+
+    position: absolute;
+    left: 0;
+    right: 0;
+    top: $list-box-height;
+    background-color: $ui-01;
+    max-height: rem(120px);
+    overflow-y: auto;
+    z-index: z('dropdown');
+  }
+
+  .bx--list-box__menu-item {
+    display: flex;
+    align-items: center;
+    font-size: rem(14px);
+    height: rem(40px);
+    padding: 0 1rem;
+    cursor: pointer;
+    user-select: none;
+  }
+
+  .bx--list-box__menu-item:hover,
+  .bx--list-box__menu-item--highlighted {
+    background-color: $brand-01;
+    color: white;
+  }
+
+  .bx--list-box__menu-icon {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    height: 100%;
+    padding: 0 1rem;
+    transition: transform 200ms $bx--standard-easing;
+    cursor: pointer;
+  }
+
+  .bx--list-box__menu-icon > svg {
+    fill: $brand-01;
+    height: 100%;
+  }
+
+  .bx--list-box__menu-icon--open {
+    transform: rotate(180deg);
+  }
+
+  .bx--list-box__selection {
+    display: inline-block;
+    position: absolute;
+    top: 0;
+    right: rem(26px);
+    bottom: 0;
+    height: rem(40px);
+    padding: 0 1rem;
+    cursor: pointer;
+    user-select: none;
+    transition: background-color 200ms $bx--standard-easing;
+  }
+
+  .bx--list-box__selection > svg {
+    fill: $ui-05;
+    height: 100%;
+  }
+
+  .bx--list-box__selection:focus {
+    outline: 1px solid $brand-01;
+  }
+
+  .bx--list-box__selection--multi {
+    position: static;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    background-color: $brand-01;
+    height: rem(18px);
+    color: white;
+    font-size: 12px;
```

Comment:
can you replace this with our type scale mixin? so in this case it would be `@include typescale('omega')`